Data/future/usa/margins/CL.csv

```diff
@@ -46,3 +46,39 @@ date,initial,maintenance
 20150708,5060,4600
 20160223,3850,3500
 20160414,3740,3400
+20160804,3520,3200
+20160831,3190,2900
+20161205,3520,3200
+20170216,3190,2900
+20170405,2970,2700
+20170622,2750,2500
+20170725,2640,2400
+20170930,2475,2250
+20171205,2310,2100
+20180105,2145,1950
+20180214,2310,2100
+20180418,2420,2200
+20180421,2630,2300
+20180511,2695,2450
+20180530,2805,2550
+20180626,3025,2750
+20180629,3245,2950
+20180713,3575,3250
+20180718,3795,3450
+20180828,3410,3100
+20181115,3905,3550
+20181123,4015,3650
+20181127,4510,4100
+20181130,4015,3650
+20181204,4702.5,4275
+20190205,4290,3900
+20190307,4070,3700
+20190329,3740,3400
+20190501,3465,3150
+20190611,3795,3450
+20190621,4015,3650
+20190918,4785,4350
+20191017,4950,4500
```

Comment:
No advisory notice is effective on Oct. 17, 2019 - maybe you meant 2019-10-23? https://www.cmegroup.com/notices/clearing/2019/10/Chadv19-351.html